src/BlockTemplatesController.php

```diff
@@ -5,6 +5,7 @@
 use Automattic\WooCommerce\Blocks\Templates\BlockTemplatesCompatibility;
 use Automattic\WooCommerce\Blocks\Templates\ProductAttributeTemplate;
 use Automattic\WooCommerce\Blocks\Utils\BlockTemplateUtils;
+use Automattic\WooCommerce\Blocks\Templates\ProductSearchResultsTemplate;
```

Comment:
Is this class used somewhere in this file? If not, do we need to `use` it?

Reply:
Removed!